### PR TITLE
Added support to WCF serialization through DataContract

### PR DESCRIPTION
**Pull request by carvrodrigo**
Supports WCF Serialization

### CODE REVIEW

File: src/DrHouse/HealthData.cs [hunk 1]
```diff
@@ -14,12 +16,16 @@ public HealthData(string appName)
...
+        [DataMember]
         public string Name { get; private set; }
 
+        [DataMember]
         public string Type { get; set; }
 
+        [DataMember]
         public string ErrorMessage { get; set; }
 
+        [DataMember]
```
Comment (mdalepiane) [inline src/DrHouse/HealthData.cs]: Please move this attribute to the property IsOk, otherwise the chain validation will not be triggered.

**Review by mdalepiane — CHANGES_REQUESTED** [review]

File: src/DrHouse/HealthData.cs [hunk 2]
```diff
@@ -14,13 +16,18 @@ public HealthData(string appName)
...
+        [DataMember(EmitDefaultValue = false)]
         public string Type { get; set; }
 
+        [DataMember(EmitDefaultValue = false)]
         public string ErrorMessage { get; set; }
-
+        
         private bool _isOk;
+
+        [DataMember(EmitDefaultValue = false)]
```
Comment (mdalepiane) [inline src/DrHouse/HealthData.cs]: This property should always be serialized. Did you check what would happen if ```IsOK``` is ```false``` and ```EmitDefaultValue = false```?

Comment (carvrodrigo) [reply src/DrHouse/HealthData.cs]: You're right. My intention was to hide any default param value but at this time check if the state is ok or not is a core for this.

**Comment by carvrodrigo** [comment]
Changes made

**Review by mdalepiane — APPROVED** [review]

**Pull request merged**